packages/nx/src/adapter/ngcli-adapter.ts

```diff
@@ -750,10 +750,12 @@ function findDeletedProjects(host: Tree): FileChange[] {
 }
 
 function findMatchingFileChange(host: Tree, path: Path) {
-  const targetPath = path.startsWith('/') ? path.substring(1) : path.toString();
+  const targetPath = normalize(
+    path.startsWith('/') ? path.substring(1) : path.toString()
+  );
   return host
     .listChanges()
-    .find((f) => f.path === targetPath && f.type !== 'DELETE');
+    .find((f) => normalize(f.path) === targetPath && f.type !== 'DELETE');
```

Comment:
Can we invert the checks (to avoid normalizing paths when `f.type === 'DELETE'`)?